### PR TITLE
feat: authorizations

### PR DESCRIPTION
**Pull request by nicosantangelo**
# Currently being tested the builder
https://github.com/decentraland/builder/compare/feat/meta-txs?expand=1

### CODE REVIEW

File: src/modules/authorization/utils.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+export function hasAuthorization(
+  authorizations: Authorization[],
+  authorizationToFind: Authorization
+) {
+  return authorizations.some(authorization =>
+    areEqual(authorization, authorizationToFind)
+  )
+}
+
+export function areEqual(left: Authorization, right: Authorization) {
```
Comment (nachomazzara) [inline src/modules/authorization/utils.ts:17]: What do you think of adding a comment about that is `right` and `left` here?

**Review by nachomazzara — COMMENTED** [review]
Excellent!

File: README.md [hunk 2]
```diff
@@ -512,6 +513,83 @@ export function invitesReducer(
 </p>
 </details>
 
+## Authorization
+
+This module allows you to grant/revoke permissions to a token. It works for both allowance and approval for all.
```
[the remark this answers is not in the document]
Comment (nicosantangelo) [reply README.md:518]: agree

**Pull request merged**